pages/linux/wifi-menu.md

```diff
@@ -0,0 +1,16 @@
+# wifi-menu
+
+> Interactively connect to a wireless network.
+> More information: <https://manned.org/wifi-menu>.
+
+- Setup a wireless connection interactively:
+
+`wifi-menu`
+
+- Interactively setup a connect to a network and obscure the password:
+
+`wifi-menu --obscure`
```

Comment:
Wait why is this an option?! It should be enabled by default.

Reply:
Because who doesn't want someone peering over their shoulder at their network password